infra/src/feature-env.ts

```diff
@@ -117,7 +117,7 @@ yargs(process.argv.slice(2))
         env,
       )
       await writeToOutput(
-        await renderHelmValueFileContent(env, habitat, featureYaml, 'no-mocks'),
+        await renderHelmValueFileContent(env, habitat, featureYaml, argv.withMocks ?? false ? 'with-mocks' : 'no-mocks'),
```

Comment:
```suggestion
        await renderHelmValueFileContent(env, habitat, featureYaml, argv.withMocks ? 'with-mocks' : 'no-mocks'),
```

`undefined` is also falsey

Reply:
True, still I prefer to make the default explicit.